### PR TITLE
Update wording on issue forms

### LINKED ISSUE #663: Setup Github issue template
13 Dec - James to update template
9 Dec - Rufaro review complete; feedback sent to James
5 Dec - pending review; aim to complete by end of week or earlier
1 Dec - sent over to Rufaro for review; work in progress
29 Nov - PR raised
28 Nov - prerequisite for contributions and support page

### PR DESCRIPTION
**Pull request by james-nash**
Closes #663

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -8,12 +8,14 @@ body:
     attributes:
       value: |
         ### ⚠️ Warning
-        This is a public repository, so **please do not include any company-internal code or details in your bug report**.
+        This is a public repository, so **please do not include any any company-internal code or sensitive data in your bug report**.
```
Comment (Fercas123) [inline .github/ISSUE_TEMPLATE/bug_report.yml:11]:
```suggestion
        This is a public repository, so **please do not include any company-internal code or sensitive data in your bug report**.
```

Comment (james-nash) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Well spotted! Thanks

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ body:
     attributes:
       value: |
         ### ⚠️ Warning
-        This is a public repository, so **please do not include any company-internal code or details in your bug report**.
+        This is a public repository, so **please do not include any any company-internal code or sensitive data in your bug report**.
```
Comment (Fercas123) [inline .github/ISSUE_TEMPLATE/feature_request.yml:11]:
```suggestion
        This is a public repository, so **please do not include any company-internal code or sensitive data in your bug report**.
```

Comment (Fercas123) [reply .github/ISSUE_TEMPLATE/feature_request.yml]: @james-nash another one, i guess to be fair, we really don't want any any company internal code

**Review by Fercas123 — APPROVED** [review]

**Pull request merged**